### PR TITLE
[HMC-85] - Adjustment Our Class

### PR DESCRIPTION
**Pull request by tjoetmuty**
## Description :

- finishing text between carousel feat and card feat

## Screenshots / Video Record:

image

### CODE REVIEW

File: src/features/Main/components/OurClassText/index.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { FC } from 'react';
+
+const OurClassText: FC = () => {
+  return (
+    <div className="flex justify-center items-center my-10 mx-4 md:mx-2 ">
+      <div className="text-wrap w-[750px]">
+        <h1 className="font-semibold text-[#00CCFF] text-center text-[20px] md:text-[30px]">
```
Comment (vickyadrii) [inline src/features/Main/components/OurClassText/index.tsx:7]: if you've got a custom color, feel free to add it to `tailwind.config.ts`, 
for example: 
```
theme: {
    extend: {
        colors: {
            "primary-blue": "#00CCFF",
         },
    }, 
}
```

Comment (mpsalunggg) [reply src/features/Main/components/OurClassText/index.tsx:7]: @tjoetmuty check the color on the tailwind config, if it already exists, use the existing color

Comment (tjoetmuty) [reply src/features/Main/components/OurClassText/index.tsx]: @vickyadrii finished 3df8a46f2b9a8c523816326fde1d9f5b305aca6e

Comment (tjoetmuty) [reply src/features/Main/components/OurClassText/index.tsx]: @mpsalunggg the custom color for it didnt exist yet, so i added it. and for button "learn more", i havent finished but im still working on it

**Review by vickyadrii — APPROVED** [review]

**Review by akwanmaroso — APPROVED** [review]

**Comment by tjoetmuty** [comment]
@mpsalunggg done 85d693c50d6eca79ccd78914eea358423a370ee4

**Review by mpsalunggg — APPROVED** [review]
yapss lets merge

**Pull request merged**